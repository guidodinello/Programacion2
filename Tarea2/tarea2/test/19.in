# Implementar hayNatsRepetidos
hayNatsRepetidos
insertarAlFinal (6,2.4)
hayNatsRepetidos
insertarAlFinal (4,2.4)
hayNatsRepetidos
insertarAlFinal (9,2.4)
hayNatsRepetidos
insertarAlFinal (9,3.4)
hayNatsRepetidos
finalCadena
cambiarEnCadena (7,2.4)
hayNatsRepetidos
insertarAlFinal (6,3.4)
hayNatsRepetidos
finalCadena
cambiarEnCadena (1,2.4)
hayNatsRepetidos
imprimirCadena
finalCadena
insertarAntes (4,3.4)
hayNatsRepetidos
imprimirCadena
Fin
