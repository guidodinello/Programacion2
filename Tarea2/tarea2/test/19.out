1>#  Implementar hayNatsRepetidos.
2>En cad no hay naturales repetidos.
3>Insertado al final.
4>En cad no hay naturales repetidos.
5>Insertado al final.
6>En cad no hay naturales repetidos.
7>Insertado al final.
8>En cad no hay naturales repetidos.
9>Insertado al final.
10>En cad hay naturales repetidos.
11>loc al final.
12>Cambio.
13>En cad no hay naturales repetidos.
14>Insertado al final.
15>En cad hay naturales repetidos.
16>loc al final.
17>Cambio.
18>En cad no hay naturales repetidos.
19>(6,2.40)(4,2.40)(9,2.40)(7,2.40)(1,2.40)
20>loc al final.
21>Insertado antes de loc.
22>En cad hay naturales repetidos.
23>(6,2.40)(4,2.40)(9,2.40)(7,2.40)(4,3.40)(1,2.40)
24>Fin.
